### DISCUSSION TITLE
sheet size when presented?

### DISCUSSION THREAD

**Discussion by ericxl** [lodev09/react-native-true-sheet#12] [Q&A] [answered]
Thanks for this awesome library!
How would I know the size of the sheet view when it is presented? I have a scrollview or web view inside the sheet and I want that view to take over the full space of the sheet all the time. Thanks!

**Comment by lodev09** [comment, marked as answer]
Hey, thanks for the feedback!

You can check [scrolling guide](https://sheet.lodev09.com/guides/scrolling) or the [`onPresent` event](https://sheet.lodev09.com/reference/props#onpresent) to get the size info.

Let me know if that helps!
